### DISCUSSION TITLE
OCI Registry for docker-registry.helm

### DISCUSSION THREAD

**Discussion by wkbrd** [twuni/docker-registry.helm#123] [Q&A]
Hi,

Is there an OCI registry where this helm chart can be pulled?  I'd prefer to pull it from an OCI registry versus the older Helm registry format?

Thanks!

**Comment by vyas-n** [comment]
Not at the moment, but it could be setup.

GitHub Packages can store Docker containers and act as an OCI Helm Chart Repo.
This process can also be automated with a GitHub Action Workflow.

@canterberry, would you mind if I worked on a proof-of-concept for this using a fork of this repo?

**Reply by canterberry** [reply]
:+1: Sure!

**Reply by wkbrd** [reply]
Hi @vyas-n , any update on the OCI Helm Chart Repo proof-of-concept?

**Reply by vyas-n** [reply]
@wkbrd My bad, I haven't been working on it lately!

Most likely it'll take sometime for me to get around to it. However, if you want to take it up, I'd be happy to review the changes.

**Reply by wkbrd** [reply]
Hi @vyas-n , please check out #141 when you have a moment. Thanks!

**Comment by wkbrd** [comment]
A candidate version of this is now available here:
https://github.com/twuni/docker-registry.helm/pull/141
